### ISSUE TITLE
Why? (Why NOT!) What? Who? When? How?

### ISSUE THREAD

**Issue by nelsonic** [dwyl/home#1]
Answer all of these questions.

**Comment by nelsonic** [comment]
Built-in brand recognition ... https://www.google.pt/search?tbm=isch&q=home+icon
![home-icon-brand-recognition](https://user-images.githubusercontent.com/194400/44060896-1e208460-9f4e-11e8-93f5-34147b7691df.png)

**Comment by nelsonic** [comment]
Do things that _don't_ scale! http://paulgraham.com/ds.html

**Comment by nelsonic** [comment]
http://paulgraham.com/95.html
![image](https://user-images.githubusercontent.com/194400/44198354-173ece80-a139-11e8-814c-2921539649c3.png)

**Comment by nelsonic** [comment]
10 things the coliving industry will need to succeed
https://coliving.com/blog/10-things-the-coliving-industry-will-need-to-succeed

Which do you _agree_/_disagree_ with...?

**Comment by nelsonic** [comment]
![image](https://user-images.githubusercontent.com/194400/44294659-aa3f4c00-a292-11e8-9b5d-650cc7de6e98.png)
https://twitter.com/KatAlexPas/status/670278026324795393
https://medium.com/founders-coders/from-non-technical-to-hired-in-5-months-d010f601b1bc

**Comment by nelsonic** [comment]
https://www.cm-braga.pt/pt/1101/viver/desporto/projetos/item/item-1-522
![image](https://user-images.githubusercontent.com/194400/44634699-65579b80-a99d-11e8-85ad-406e572d33d2.png)

**Comment by nelsonic** [comment]
# A Cautionary Tale ~ Artist-Run Space?

https://www.poetryfoundation.org/harriet/2018/04/so-you-want-to-open-a-small-press-bookstore-artist-run-space-a-cautionary-tale

> "Unless you’re “independently wealthy,” you probably don’t have enough money to open a small press bookstore or artist-run space, which means you’ll probably have to be a **non-profit**, which is almost guaranteed to be a **terrible idea** for a number of reasons."

discussion: https://news.ycombinator.com/item?id=18261895

**Comment by iteles** [comment]
This led me down a [Ghost Ship Fire](https://en.wikipedia.org/wiki/2016_Oakland_warehouse_fire) rabbit hole, where the leaseholders (not the owners) of the building were arrested and are serving up to 9 and 6  years in jail respectively. Wow. Sad times.

**Comment by nelsonic** [comment]
@iteles yep ... being **_personally_ liable** for **fire safety** is both a _good_ thing to keep people accountable/responsible and a _terrifying_ thing for people starting/running a communal space. 😞 

As much as I know people will accuse me of being a "**party pooper**" for being unwaveringly against 
Alcohol, Drugs and (overnight) Parties, I know that it's the _right_ decision for the **_long_-term** 
because we want to build a _sustainable_ and _safe_ creative community, not a "frat house" where people come for one week to take [psychedelics](https://en.wikipedia.org/wiki/Psychedelic_drug) 🦄, have "relations" (_the fallout of which will_ ["_**sink** the ship_"!](https://github.com/dwyl/home/tree/0435d8db7626030b9d923350645070b39276ce39#why-it-will-never-work-risks-and-mitigation)) and burn the place down! 🔥

We will have _plenty_ of fun including social gatherings, playing music (_at [reasonable volume](https://www.quora.com/What-is-the-best-volume-to-listen-to-music) ..._), 
film nights and regular [vegan BBQs](https://github.com/dwyl/health/issues/88) and if people want to _go_ out and _party_ 💃🕺🎉  
they are welcome to provided they are respectful of the people who are sleeping (_because they have goals/routines that require [discipline](https://youtu.be/C-Cvl3_CH2A?t=386)_) and don't set off the noise alarms 🔇  when they **`return`**.

**Comment by nelsonic** [comment]
https://youtu.be/NB3e8xwu_Do
